corehq/apps/sso/utils/entra.py

```diff
@@ -96,7 +96,8 @@ def _get_user_principal_names(user_ids, token):
 
         # Extract userPrincipalName from batch response
         for resp in batch_result['responses']:
-            if 'body' in resp and 'userPrincipalName' in resp['body']:
+            if ('body' in resp and 'userPrincipalName' in resp['body']
+            and resp['body'].get('accountEnabled') is True):
```

Comment:
`will `accountEnabled` actually return a boolean or will it be the string `true`/`false`? did you test this locally?

Reply:
It is a boolean. I tested locally and on staging.